### PR TITLE
Tweak the readme to be more user facing

### PR DESCRIPTION
**Pull request by hadley**
Should I also setup keyring.r-lib.org?

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -13,108 +13,74 @@ knitr::opts_chunk$set(
...
 Install the package from CRAN:
 
 ```{r eval = FALSE}
-install.packages("keyring")
+# install.packages("pak")
+pak::pak("keyring")
 ```
 
-## Usage
+We recommend using pak to install keyring as it will ensure that all Linux system requirements (e.g. `libsecret-devel`, `openssl-devel`) are automatically installed.
```
Comment (gaborcsardi) [inline README.Rmd:46]: If we are mentioning them already, we can maybe mention `libsodium-devel` as well, and maybe also note that these are for (say) Fedora 38.
```
❯ pak::pkg_sysreqs("keyring", sysreqs_platform = "fedora-38")
── Install scripts ──────────────────────────────────────────────── Fedora 38 ──
dnf install -y libsecret-devel openssl-devel libsodium-devel

── Packages and their system dependencies ──────────────────────────────────────
keyring – libsecret-devel
openssl – openssl-devel
sodium  – libsodium-devel
```

```
❯ pak::pkg_sysreqs("keyring", sysreqs_platform = "ubuntu-2204")
── Install scripts ────────────────────────────────────────────── Ubuntu 2204 ──
apt-get -y update
apt-get -y install libsecret-1-dev libssl-dev libsodium-dev

── Packages and their system dependencies ──────────────────────────────────────
keyring – libsecret-1-dev
openssl – libssl-dev
sodium  – libsodium-dev
```

**Review by gaborcsardi — COMMENTED** [review]
Thanks, looks nice! Just has some tiny comments.

File: README.Rmd [hunk 2]
```diff
@@ -13,108 +13,74 @@ knitr::opts_chunk$set(
...
-Should you need to change the default backend, set the
-`R_KEYRING_BACKEND` environment variable or the `keyring_backend` R
-option to the backend's name (e.g. `env`, `file`, etc.).
+Each secret is associated with a keyring.
+By default, keyring will use the OS keyring (see `default_backend()` for details), which is automatically unlocked you log in.
+That means while the secret is stored securely, it can be accessed by other processes.
 
-### Manage keyrings:
+If you want greater security you can create a custom keyring that you manually lock and unlock.
+That will require you to enter your system password every time you want to access your secret.
```
[the remark this answers is not in the document]
Comment (hadley) [reply README.Rmd:68]: Does it default to the system password or do you need to supply it every time?

[the remark this answers is not in the document]
Comment (gaborcsardi) [reply README.Rmd:68]: You need to supply a password when you create the keyring. In general it is not possible to reuse the system password, I think.

**Comment by gaborcsardi** [comment]
> Should I also setup keyring.r-lib.org?

```
❯ dig keyring.r-lib.org | grep CNAME
keyring.r-lib.org.	300	IN	CNAME	r-lib.github.io.
```

It is set up, I'll change the config in a sec.

**Comment by gaborcsardi** [comment]
Thank you!

**Pull request merged**